docs/topics/kubernetes-developers.md

```diff
@@ -123,3 +123,48 @@ AZURE_CLOUD_NODE_MANAGER_IMG=myrepo/my-cnm:v0.0.1 \
 CLUSTER_TEMPLATE=cluster-template-external-cloud-provider.yaml \
 make create-workload-cluster
 ```
+
+## Testing clusters built from Kubernetes source
+
+[A template is provided](../../templates/cluster-template-custom-builds.yaml) that enables building clusters from custom built Kubernetes components. At this time only tilt flows have been tested. Here's an example of a `tilt-settings.json` suitable for testing from custom-built artifacts:
```

Comment:
looks like you moved this template to test/ 

That's good because we don't want the custom build template to be part of release artifacts but it would make sense to enable it on tilt for dev purposes. Maybe we can add it to the list of templates tilt uses for local resources somehow? https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/master/Tiltfile#L226